packages/docs/docs/communications/async-encounters.md

```diff
@@ -0,0 +1,53 @@
+
+You should record the participating physicians using the `Encounter.participant` element. You can also record any family members who are part of the session here (see our [Family Relationships guide](#) ).
+
+:::tip Asynchronous Encounter Ontologies
+
+The `Encounter.class` is a required element in FHIR and should be taken from the [HL7 Act Encounter Code Valueset](https://terminology.hl7.org/3.1.0/ValueSet-v3-ActEncounterCode.html). Asynchronous care contexts will almost always use the code `VR` ("virtual").
+
+Also check our [USCDI guide](/docs/fhir-datastore/understanding-uscdi-dataclasses) for information on how to make your [Encounter](/docs/api/fhir/resources/encounter) compatible with the US Core standards.
+
+:::
+
+If your session only involves providing care for a single patient, then you can set the `Encounter.subject` element to refer to the patient and you're all set! If, however, multiple patients are involved in the session, continue reading.
+
+## Handling Multiple-Patient Sessions
+
+In some care settings, a session may discuss the health of multiple patients. For example, a mother may ask about the health of both of her children in the same email exchange.
+
+In these situations, we'll have to represent distinct "medical encounters" for each patient.
+
+We can use the hierarchical nature of the [Encounter](/docs/api/fhir/resources/encounter) resource to split out medical encounters for each session. The `Encounter.partOf` element creates a parent-child relationship between [Encounters](#), which is perfect for encounters that overlap in time.
+
+To properly represent your asynchronous encounter, you should:
+
+1. Create an [Encounter](/docs/api/fhir/resources/encounter) for the session, [as described above ](#representing-sessions-in-fhir)
+2. Create a new [Encounter](/docs/api/fhir/resources/encounter) for each patient to represent a medical encounter
+3. Set the `Encounter.subject` of each medical encounter to the corresponding patient
+4. Populate each medical encounter with the clinical details (diagnoses, reasons for visit) of the corresponding patient.
+5. Set the `Encounter.partOf` element for each medical encounter to refer to the session's [Encounter](#)
+
+The [Communication](#) resources should still be linked to the "session" encounter, but the clinical details for each patient will live on the medical encounters. You can add an additional value the `Encounter.type` element to tag encounters as either "sessions" or "medical encounters."
+
+While creating an [Encounter](/docs/api/fhir/resources/encounter) hierarchy like this is a bit more work up front, it promotes good data hygiene. Separating out medical details for each patient allows for better patient analytics and also helps with [billing](/docs/billing), as payer systems typically expect a single encounter per patient.
```

Comment:
I'd add a bit stronger language here.  A well documented encounter, with the correct Practitioner, Diagnosis codes, service type is critical to correct billing.  For reference, this is a sample encounter for USCDI

```
{
  "id": "82bbbd9a-f7af-4ba3-a55d-7c4255c3bfa1",
  "meta": {
    "profile": [
      "http://hl7.org/fhir/us/core/StructureDefinition/us-core-encounter"
    ],
    "versionId": "8f8e2dc5-9a1f-4a48-91d1-da136d1f5d66",
    "lastUpdated": "2023-01-22T19:24:17.575Z",
    "author": {
      "reference": "Practitioner/ff3139bc-fe17-416b-8016-8285498e9c8c",
      "display": "Reshma Khilnani"
    },
    "project": "11ba3e91-e233-4e27-9a1c-bafdbb34061f",
    "compartment": [
      {
        "reference": "Project/11ba3e91-e233-4e27-9a1c-bafdbb34061f"
      },
      {
        "reference": "Patient/13e44a47-636b-49e2-adb3-9f19c7e0e47a",
        "display": "Mr. Dustin31 Ritchie586"
      }
    ]
  },
  "identifier": [
    {
      "use": "official",
      "system": "https://github.com/synthetichealth/synthea",
      "value": "a2590942-80e3-b27b-39ab-017a88b3fed1"
    }
  ],
  "status": "finished",
  "class": {
    "system": "http://terminology.hl7.org/CodeSystem/v3-ActCode",
    "code": "AMB"
  },
  "type": [
    {
      "coding": [
        {
          "system": "http://snomed.info/sct",
          "code": "162673000",
          "display": "General examination of patient (procedure)"
        }
      ],
      "text": "General examination of patient (procedure)"
    }
  ],
  "subject": {
    "reference": "Patient/13e44a47-636b-49e2-adb3-9f19c7e0e47a",
    "display": "Mr. Dustin31 Ritchie586"
  },
  "participant": [
    {
      "type": [
        {
          "coding": [
            {
              "system": "http://terminology.hl7.org/CodeSystem/v3-ParticipationType",
              "code": "PPRF",
              "display": "primary performer"
            }
          ],
          "text": "primary performer"
        }
      ],
      "period": {
        "start": "2020-10-15T19:33:18-04:00",
        "end": "2020-10-15T19:48:18-04:00"
      },
      "individual": {
        "reference": "Practitioner/bbb136d1-88af-4dab-b60b-beabc8ec558a",
        "display": "Dr. Blossom971 Christiansen251"
      }
    }
  ],
  "period": {
    "start": "2020-10-15T19:33:18-04:00",
    "end": "2020-10-15T19:48:18-04:00"
  },
  "location": [
    {
      "location": {
        "reference": "Location/e97cda0e-ab8b-436a-a79c-103bdc855c09",
        "display": "PCP170967"
      }
    }
  ],
  "serviceProvider": {
    "reference": "Organization/89fdb7b6-b113-4831-b382-4f07eb7222cc",
    "display": "PCP170967"
  },
  "resourceType": "Encounter"
}
```